### DISCUSSION TITLE
How to include Legrand 412172 kwH meter device

### DISCUSSION THREAD

**Discussion by hocomadvies** [Koenkk/zigbee2mqtt#24847] [Devices]
Hi, I have a Legrand 412172 installed to read some kw~H from the heatpump 1 fase. How to get that included into zigbee2mqtt? it does not show any light blinking? I see there is a small push button on it. 

I'm on version [1.28.0]

**Comment by chris-1243** [comment]
You need to check in the user manual how to put your device in pairing mode.

Indeed, it is supported. Keep in mind, the devices list published on the website (zigbee2mqtt.io) refers to the latest version. 

You are on 1.28.0
Latest is 1.41.0

I would update first to ensure full support of your device in order to avoid an `unsupported device` state while pairing.
